packages/router/src/page-loader.tsx

```diff
@@ -103,6 +103,10 @@ export class PageLoader extends React.Component<Props> {
     }
   }
 
+  componentWillUnmount() {
+    this.setState = () => {} // Prevent updating state after component has been unmounted.
+  }
+
```

Comment:
React was trying to update state on unmounted component in the tests for some reason. I tried running it down but didn't have a ton of time. This was put as a placeholder. Can circle later

Reply:
Ok, I think we're going to remove the delayed page loading stuff which makes this code super difficult to reason about.